### ISSUE TITLE
the hangup button on 1:1 voip calls can fade away while you're hovering over it

### ISSUE THREAD

**Issue by ara4n** [vector-im/element-web#16751]
If you stop moving the mouse for a while when hovering over the hangup button, it can fade out.  Which is a bit frustrating, given you then have to move away and back again to make it reappear, meaning you hang up a few seconds later than you intended and look like a divot

**Comment by SimonBrandner** [comment]
Closing in favour of https://github.com/vector-im/element-web/issues/18221

**Comment by ara4n** [comment]
https://github.com/matrix-org/matrix-react-sdk/pull/6472/commits/17546026133259874c60fd29c02c65f2d415e9fc doesn't fix this.  We should never be fading away buttons when the user has put their mouse over them (whether that's after 1s or 2s).

**Comment by SimonBrandner** [comment]
> [matrix-org/matrix-react-sdk@1754602](https://github.com/matrix-org/matrix-react-sdk/commit/17546026133259874c60fd29c02c65f2d415e9fc) doesn't fix this. We should never be fading away buttons when the user has put their mouse over them (whether that's after 1s or 2s).

The PR should have fixed this, will check on develop

**Comment by SimonBrandner** [comment]
Yeah, looks fixed to me on develop, scream if it's still broken for you

**Comment by t3chguy** [comment]
Yeah indeed they don't vanish for me on develop either when hovering directly on the button.
